frontend/src/employee-frontend/components/reports/AttendanceReservationByChild.tsx

```diff
@@ -249,6 +252,18 @@ export default React.memo(function AttendanceReservationByChild() {
           />
         </FilterRow>
 
+        {featureFlags.aromiIntegration && roles.includes('ADMIN') && (
```

Comment:
ok if this is a temporary functionality until integration, otherwise wouldn't hard code roles here